README.md

```diff
@@ -186,7 +186,9 @@ I would like to thank:
 This project is [MIT](./LICENSE.md) licensed.
 
 _NOTE: we recommend using the [MIT license](./LICENSE.md) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+ feature3/validations-specs
 
 _NOTE: we recommend using the [MIT license](https://github.com/techEmprez/Blog_app/new/dev) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
 
```

Comment:
- [ ] [ OPTIONAL ] your requests tests are failing it would be better to make the pass
<img width="716" alt="Screenshot 2023-01-20 at 16 18 59" src="https://user-images.githubusercontent.com/61790587/213734404-c98fc090-062f-4c6c-a331-1b9d09612bff.png">